app/controllers/user/courses_controller.rb

```diff
@@ -1,8 +1,8 @@
 class User::CoursesController < User::ApplicationController
+  load_and_authorize_resource :course, class: "Course",
```

Comment:
```suggestion
  load_and_authorize_resource :course, class: Course.name,
```
Tương tự cho các class khác